examples/README.md

```diff
@@ -1,53 +0,0 @@
-Sample OAuth 2.0 servers and clients
```

Comment:
Yes, a search by a JIRA label of "RetiredCode" or by a keyword of "in memory data store" will find it - https://issues.gpii.net/browse/GPII-2055